### PR TITLE
Create controllers

### PR DESCRIPTION
**Pull request by ekenecf**
For each URL I create:

A route.
Action in the correct controller.
A view file.
View files should include only placeholders written in plain HTML, e.g:

Here is a list of posts for a given user
Make sure that you added code to handle only the 4 required URLs.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
```
Comment (TedLivist) [inline app/controllers/posts_controller.rb:4]: - [ ] You are missing the accompanying post show view: `app/views/posts/show.html.erb`. Kindly add the file as well

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 class UsersController < ApplicationController
-  def hello; end
+  def index; end
+
+  def show; end
```
Comment (TedLivist) [inline app/controllers/users_controller.rb:4]: - [ ] You are missing the accompanying user show as well: `app/views/users/show.html.erb`. Kindly add that file.

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+  get 'comments/index'
+  get 'likes/index'
+  get 'posts/index'
+  root to: 'users#index'
+
+  resources :users, only: %i[index show] do
+    resources :posts, only: %i[index new ] do
+      resources :likes, only: %i[new ]
+    resources :comments, only: %i[new ]
+    end
```
Comment (TedLivist) [inline config/routes.rb]: - [ ] It is good practice to only expose needed routes. We don't really need to expose all these routes yet. Kindly remove unnecessary routes. We can have something like so:

  ```
  root to: 'users#index'

  resources :users, only: %i[index show] do
    resources :posts, only: %i[index new ]
  end
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ekenecf,

Good job so far!
✔️ Good documentation
✔️ Creation of index actions and views for posts and users
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (rloterh) [reply app/controllers/posts_controller.rb:4]: This issue is well resolved! ✔️

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (rloterh) [reply app/controllers/users_controller.rb:4]: This issue is well resolved! ✔️

File: config/routes.rb (hunk 3 above)
Comment (rloterh) [reply config/routes.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ekene,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Project is set up correctly


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**